File: sum_of_multiples_of_3_or_5_under_1000/sum_of_multiples_of_3_or_5.py
def main():
    sum = 0
    for i in range(1, 1000):
        if i % 3 == 0 or i % 5 == 0:
            sum = sum + i
    print("The sum of the natural numbers below 1000 that are multiples of 3 or 5 is %d!" %
          sum)

if __name__ == "__main__":
    main()

File: sum_of_multiples_of_3_or_5_under_1000/sum_of_multiples_of_3_or_5.c
# include <stdio.h>

int main() {
    int sum = 0
    int i
    for (i=0
         i < 1000
         i + +) {
        if (i % 3 == 0 | | i % 5 == 0) {
            sum = sum + i
        }
    }
    printf(
        "The sum of the natural numbers below 1000 that are multiples of 3 or 5 is %d!\n", sum)
    return 0
}
